src/ocean/util/log/Config.d

```diff
@@ -315,7 +315,9 @@ public Layout newLayout ( istring layout_str )
             break;
 
         default:
-            throw new Exception("Invalid layout requested : " ~ layout_str);
+            istring msg = "Invalid layout requested : ";
```

Comment:
The change was "necessary" because `istring ~ cstring` yields `cstring` (stupid, I know). Thus `throw new Exception("..." ~ layout_str);` stopped to compile after I changed `layout_str` to `cstring`.
The options, were to add a `cast` (unnecessary and prone to refactoring errors), use `format` (adding a new dependency seemed overkill), or just break down the statement. And since I was changing those line, I figured I might just add `__FILE__` and `__LINE__`.